pillar.example

```diff
@@ -45,3 +45,7 @@ zendserver:
   # Directives to be set or modified
   #directives:
   #  - zray.zendserver_ui_url: "https://your.server.example.com:10082/ZendServer"
+
+  # Job queue daemon settings
+  jqd:
+    http_retry_count: '0'
```

Comment:
Maybe we should default it to 10, so the default behaviour doesn't change? i don't know how many other projects are using this formula. We can override this with '0' in our BLL pillar I think.

Reply:
Resolved by c40f4fa058475dede96d221c0c0bebd1af148647